kirby.php

```diff
@@ -138,11 +141,14 @@ public function defaults() {
       // auto template css files
       if($url == '@auto') {
 
-        $file = $kirby->site()->page()->template() . '.css';
-        $root = $kirby->roots()->autocss() . DS . $file;
-        $url  = $kirby->urls()->autocss() . '/' . $file;
+        $file    = $kirby->site()->page()->template() . '.css';
+        $root    = $kirby->roots()->autocss() . DS . $file;
+        $url     = $kirby->urls()->autocss() . '/' . $file;
 
-        if(!file_exists($root)) return false;
+        if(!file_exists($root)) {
+          if(!($url = $kirby->modules()->getAsset('autocss', $file)))
+          return false;
```

Comment:
I think this should be in the same line as the `if`. Could become a new goto fail bug otherwise. ;)
Alternatively use curly braces.

Same for the JS handler.